config/opal_config_pthreads.m4

```diff
@@ -281,11 +281,11 @@ if test "$opal_pthread_c_success" = "0"; then
                               opal_pthread_c_success=0)
     AC_LANG_POP(C)
     if test "$opal_pthread_c_success" = "1"; then
-      PTHREAD_CFLAGS="$pf"
+      TPKG_CFLAGS="$pf"
```

Comment:
What is `TPKG`?  Why is it better than a `PTHREAD` prefix?  The tests here are specific to pthreads, after all...?

Reply:
@hppritcha can explain the motivation better than I can, but I believe that the TPKG_CFLAGS get used for every possible thread package.

Reply:
@npe9 is correct.  TPK_CFLAGS is used on opal_config_threads.m4.  Right now the argo and qthreads configure functions don't use it, but the name was to indicate its more general and may be set by other thread package configure functions.

Reply:
well we moved all of the threads related configury into the framework, so probably this comment doesn't apply anymore?